### DISCUSSION TITLE
Pytorch Dataloader Cant pickle Open3D Objects

### DISCUSSION THREAD

**Discussion by tsuffb** [isl-org/Open3D#4760] [Q&A]
Hey, I've built a dataset of images that are generated using open3d tools.
Some open3d objects are in the dataset constructor, objects such as: TriangleMesh, PinholeCamera etc...

When iterating over the DataLoader based on the above dataset, I get TypeErrors: 
1. TypeError: can't pickle open3d.cpu.pybind.geometry.TriangleMesh objects
2. TypeError: can't pickle open3d.cpu.pybind.camera.PinholeCameraIntrinsic objects

Full Error Prefix:
Traceback (most recent call last):
  File "C:\Users\admin\AppData\Local\Programs\Python\Python37\lib\multiprocessing\popen_spawn_win32.py", line 89, in __init__
    reduction.dump(process_obj, to_child)
  File "C:\Users\admin\AppData\Local\Programs\Python\Python37\lib\multiprocessing\reduction.py", line 60, in dump
    ForkingPickler(file, protocol).dump(obj)

What is the best way to overcome that?
Thanks

**Comment by Qiuyanghang** [comment]
I'm experiencing a similar problem, please tell me how to solve it.
Thank you so much！
